dlrs/main/lwc/rollupEditor/rollupEditor.html

```diff
@@ -1,269 +1,542 @@
 <template>
-  <lightning-card title="Lookup Rollup Summary Creation Wizard">
-    <div slot="actions">
-      <lightning-button-group>
-        <lightning-button label="Save" onclick={runSave}></lightning-button>
-        <lightning-button
-          label="Cancel"
-          onclick={cancelClickHandler}
-        ></lightning-button>
-        <lightning-button
-          label="Activate"
-          onclick={activateClickHandler}
-          if:true={rollupCanBeActivated}
-        ></lightning-button>
-        <lightning-button
-          label="Deactivate"
-          onclick={deactivateClickHandler}
-          if:true={rollupCanBeDeactivated}
-        ></lightning-button>
-        <lightning-button
-          label="Clone"
-          onclick={cloneClickHandler}
-          if:true={rollup.Id}
-        ></lightning-button>
-        <lightning-button
-          label="Delete"
-          onclick={deleteClickHandler}
-          variant="destructive"
-          if:true={rollup.Id}
-        ></lightning-button>
-      </lightning-button-group>
-    </div>
-    <div class="slds-card__body slds-card__body_inner">
-      <!--Name Rollup-->
-      <c-rollup-editor-error errors={errors.record}></c-rollup-editor-error>
-      <div class="slds-box">
-        <lightning-layout class="slds-p-horizontal_medium">
-          <lightning-layout-item size="12" class="slds-form-element_stacked">
-            <div class="slds-text-heading_small slds-border_bottom">
-              1. Name your Rollup
-            </div>
-          </lightning-layout-item>
-        </lightning-layout>
-        <lightning-layout class="slds-p-horizontal_medium">
-          <lightning-layout-item size="6" class="slds-form-element_stacked">
-            <lightning-input
-              type="text"
-              name="rollup_label"
-              data-name="rollup_label"
-              label="Lookup Rollup Summary Label"
-              placeholder="My Rollup Name"
-              value={rollup.Label}
-            ></lightning-input>
-
-            <c-rollup-editor-error
-              errors={errors.Label}
-            ></c-rollup-editor-error>
-          </lightning-layout-item>
-          <lightning-layout-item size="6" class="slds-form-element_stacked">
-            <lightning-input
-              type="text"
-              name="rollup_DeveloperName"
-              data-name="rollup_DeveloperName"
-              label="Lookup Rollup Summary API Name"
-              placeholder="My_Rollup_Name"
-              disabled={rollup.Id}
-              value={rollup.DeveloperName}
-            ></lightning-input>
-            <c-rollup-editor-error
-              errors={errors.DeveloperName}
-            ></c-rollup-editor-error>
-          </lightning-layout-item>
-        </lightning-layout>
+
+  <div class="slds-m-bottom_medium">
```

Comment:
This looks like a lot of work, thanks!

I think it looks really clean. I had a request thoughts/questions/requests along the way. Let me know your thoughts on any of it.